model/src/channel/message/mod.rs

```diff
@@ -101,23 +115,44 @@ pub struct Message {
     ///
     /// [Message Content Intent]: crate::gateway::Intents::MESSAGE_CONTENT
     pub embeds: Vec<Embed>,
+    /// Enabled [`MessageFlags`].
     #[serde(skip_serializing_if = "Option::is_none")]
     pub flags: Option<MessageFlags>,
+    /// ID of the [`Guild`] the message was sent in.
+    ///
+    /// [`Guild`]: crate::guild::Guild
     #[serde(skip_serializing_if = "Option::is_none")]
     pub guild_id: Option<Id<GuildMarker>>,
+    /// Id of the message.
     pub id: Id<MessageMarker>,
+    /// Interaction the message was sent as a response to.
```

Comment:
accidentally deleted my suggestion instead of resolving this conversation but this is good for now, though Discord's docs could be improved here language wise, and we can reflect it later